content/articles/factors_that_affects_the_performance_of_system_using_deep_learning/index.md

```diff
@@ -0,0 +1,92 @@
+In its earliest layers, GPSA appears to initialize more hyperparameters. The final (later) levels are more concerned with content data (other distinctive data sets).
+ 
+#### 3. Enough degrees of freedom 
+
+The main challenge here is fine-tuning the deep network's hyperparameters. This is a difficult task because the output's accuracy is in question. 
+What key elements are used to determine a network's degree of freedom? 
+Basic statistical speculation postulates exist to help examine the likelihood rate of an event. Larsen et al. (2021) suggest a theory to aid in the resolution of the fine-tuning issue. The probability of success in a deep network underpins this idea. This is what is expected to happen when the hyperparameters of a system data set are being trained.
+In a deep network, the degree of freedom determines whether or not success (accuracy) is possible. During the training process, the theory has aspects to consider. Gordon's Escape Theorem is generalized by the theory that establishes a powerful theorem. This theorem can be applied to any set that is well-known. In a successful chance, the principal outcome emphasizes the presence of a section transition. It's best to keep dimensionality as low as possible while initializing parameters. Dimensionality influences the sub-level set of network layers.
+The theory (lottery subspaces) seeks to repeat data from prior training experiments. It then establishes the settings for a low-dimensional projection. It compares the parameters to such a trajectory's top-dimensional principal components. The result of this theoretical hypothesis is that both postulates are true. For the same compression, subspace-confined neural networks could even outperform lottery tickets. 
+
+ ### CONCULSION
+With the vast and rapid development of technology, deep learning is the key to today’s smart world. This has brought an ease in communication between humans and their daily environment. There are some difficulties in understanding the deep learning system algorithm.
+Some real-life postulates are cited to help design the deep network algorithm.
+
+There are some factors that affect the performance of  deep learning systems. These factors tend to reduce the accuracy level of the system: These factors include: 
+* Considered a problem.
+* Evaluation difficulties of default hyperparameters for multiple optimisers.
+* Technical know-how
+* Which optimiser suits the learning algorithm best?
+
+There are some steps to aim at in order to achieve a perfect performance rate in a deep learning system. Factors like;
+* The functionality of individual layers in the deep network system.
+* The efficiency of soft convolutional inductive biases in vision transformation improvement.
+* Suitable degrees of freedom to aim at in the deep network while training.
+
+
+### Diagram Demonstrating The Different Scenarios One Can Fall Into When Configuring The Learning Rate.
```

Comment:
Is this subtopic incomplete? Why is this after conclusion? Have you missed out something?